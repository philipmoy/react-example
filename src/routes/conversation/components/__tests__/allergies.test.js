import React from 'react';
import { shallow } from 'enzyme';

import Allergies from '../allergies';

describe('ConversationAllergies', () => {
  it('renders correctly', () => {
    const wrapper = shallow(
      <Allergies />,
    );
    expect(wrapper).toMatchSnapshot();
  });
});
