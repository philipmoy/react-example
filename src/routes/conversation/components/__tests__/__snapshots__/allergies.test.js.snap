// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ConversationAllergies renders correctly 1`] = `
<div
  className="allergies"
>
  <div
    className="cell"
    key="0"
  >
    <div
      className="cell__icon"
    >
      ICON
    </div>
    <p
      className="cell__label"
    >
      Allergy
    </p>
  </div>
  <div
    className="cell"
    key="1"
  >
    <div
      className="cell__icon"
    >
      ICON
    </div>
    <p
      className="cell__label"
    >
      Allergy
    </p>
  </div>
  <div
    className="cell"
    key="2"
  >
    <div
      className="cell__icon"
    >
      ICON
    </div>
    <p
      className="cell__label"
    >
      Allergy
    </p>
  </div>
  <div
    className="cell"
    key="3"
  >
    <div
      className="cell__icon"
    >
      ICON
    </div>
    <p
      className="cell__label"
    >
      Allergy
    </p>
  </div>
  <div
    className="cell"
    key="4"
  >
    <div
      className="cell__icon"
    >
      ICON
    </div>
    <p
      className="cell__label"
    >
      Allergy
    </p>
  </div>
  <div
    className="cell"
    key="5"
  >
    <div
      className="cell__icon"
    >
      ICON
    </div>
    <p
      className="cell__label"
    >
      Allergy
    </p>
  </div>
  <div
    className="cell"
    key="6"
  >
    <div
      className="cell__icon"
    >
      ICON
    </div>
    <p
      className="cell__label"
    >
      Allergy
    </p>
  </div>
  <div
    className="cell"
    key="7"
  >
    <div
      className="cell__icon"
    >
      ICON
    </div>
    <p
      className="cell__label"
    >
      Allergy
    </p>
  </div>
  <div
    className="cell"
    key="8"
  >
    <div
      className="cell__icon"
    >
      ICON
    </div>
    <p
      className="cell__label"
    >
      Allergy
    </p>
  </div>
  <div
    className="cell"
    key="9"
  >
    <div
      className="cell__icon"
    >
      ICON
    </div>
    <p
      className="cell__label"
    >
      Allergy
    </p>
  </div>
</div>
`;
